src/app/window-handler.ts

```diff
@@ -144,16 +144,20 @@ export class WindowHandler {
         this.spellchecker = new SpellChecker();
         logger.info(`window-handler: initialized spellchecker module with locale ${this.spellchecker.locale}`);
 
+        logger.info('window-handler: createApplication mainWinPos: ' + JSON.stringify(this.config.mainWinPos));
```

Comment:
I think you don't need to  do JSON.stringify here? Just doing ${this.config.mainWinPos} should work?

Reply:
@mattias-symphony `mainWinPos` is an object, so, stringily is required actually.